cmd/list.go

```diff
@@ -39,6 +39,24 @@ var listCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		// Filter mods by side
+		if viper.IsSet("list.side") {
+			side := viper.GetString("list.side")
+			if len(side) == 0 || (side != core.UniversalSide && side != core.ServerSide && side != core.ClientSide) {
+				fmt.Println("Invalid side!")
```

Comment:
Could you list the possible options the user has? e.g. `Invalid side "front", must be one of client, server, or both (default)`